src/main/java/com/spartronics4915/lib/lidar/LidarProcessor.java

```diff
@@ -259,25 +259,31 @@ private void processLidarScan(LidarScan scan)
     {
```

Comment:
@dbadb Please make sure everything here is correct. If it's not then it's fine for you to go and fix it.

All of the other changes can be reviewed, but this is really the important one for you to look at.